.github/actions/label-released-issues-v1/src/getIssueProjectInfo.ts

```diff
@@ -0,0 +1,107 @@
+import { getOctokit } from '@actions/github'
+
+interface getIssueProjectInfoArgs {
+  owner: string
+  repo: string
+  issueNumber: number
+  octokit: ReturnType<typeof getOctokit>
+}
+
+interface Node {
+  id: string
+  type: string
+  project: {
+    number: number
+  }
+  fieldValueByName: {
+    name: string
+  }
+}
+
+/**
+ * Example of a GraphQL query to get the project info for an issue
+ * @example
+ * ```json
+ * {
+ * "data": {
+ *   "repository": {
+ *     "issue": {
+ *       "projectItems": {
+ *         "nodes": [
+ *           {
+ *             "id": "PVTI_lADOAD55W84AVmLazgJbJGI",
+ *             "type": "ISSUE",
+ *             "project": {
+ *               "number:" 104,
+ *             },
+ *             "fieldValueByName": {
+ *               "name": "Backlog"
+ *             }
+ *           }
+ *         ]
+ *       }
+ *     }
+ *   }
+ *  }
+ * }
+ * ```
+ */
+export interface GetIssueProjectInfoResult {
+  repository: {
+    issue: {
+      projectItems: {
+        nodes: Node[]
+      }
+    }
+  }
+}
+
+export default async function getIssueProjectInfo({
```

Comment:
Can't we just use or replace what we have already https://github.com/dequelabs/axe-api-team-public/tree/main/.github/actions/label-and-move-released-issues-v1/src? 

This is a 1:1 copy paste, we should avoid doing that IMO